code/game/atoms_movable.dm

```diff
@@ -173,9 +173,10 @@
 			return
 	if(A == loc && pulling.density)
 		return
-	if(!Process_Spacemove(get_dir(pulling.loc, A)))
+	var/dir = get_dir(pulling.loc, A)

```

Comment:
\>naming a local proc var the same as a src var
*cries in DM*

Reply:
we shadowing now bois